submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+ caption='Processing coal into solid fuel with heavy oil cracking, at minimum power consumption'
+
+%}
+
+The total power consumption drops from about 3.28MJ to about 0.66MJ, which translates to the energy gains of 116%, compared to the default case of 103%. This increase is nice but it is not that large because processing energy was already relatively small. Now let us look at the same example with 3 productivity 3 modules instead to see the effects of maximum productivity:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at maximum productivity'
+ caption='Processing coal into solid fuel with heavy oil cracking, at maximum productivity'
+
+%}
+
+We see that with the cumulative effects of 30% extra product output across the process results in more than double the amount of solid fuel as before! This adds up to an extra energy gain of nearly 100MJ! However, we also see that total power consumption has greatly increased to about 50.6MJ. Nevertheless, our overall energy gains have become `185.6 - 50.6 - 40 - 1.5 = 93.5MJ`, which is about 134% of energy gains instead of the default case of 103%. Meanwhile, our machines are working at half the speed as before, which makes power get consumed for twice as long.
+
+Hence finally, let's try out the compensating effects of speed modules by switching to 2 productivity 3 modules and 1 speed 3 module each:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at high productivity + speed'
+ caption='Processing coal into solid fuel with heavy oil cracking, , at high productivity + speed'
+
+%}
+
+We now find numbers somewhere in the middle of the previous two cases. We have about 12.2 solid fuel now, which is a good gain and it gives about 146.4MJ. We see that the total power consumption is now about 20.3MJ. Now, our overall energy gains have become `146.4 - 20.3 - 40 - 1.5 = 84.6MJ`, which is about 112% of energy gains instead of the default case of 103%. We discover that full productivity performs better in this case despite the machines slowing down so much.
+
+Seeing the strong effects of productivity modules, I wonder if we can make rocket fuel production yield net energy gains too, so let's fill the assembling machine 2 productivity 3 modules:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing solid fuel into rocket fuel at maximum productivity AM2'
+ caption='Processing solid fuel into rocket fuel at maximum productivity AM2'
+
+%}
+
+Well, the input and output have the same energy now, but the processing energy has grown big enough to wipe all the fuel energy gains. Looking at the cause of this high power cost, we see that it is because this recipe takes 30 seconds to craft while before we were working with 2 or 5 seconds. This difference means that a higher power consumption will has a much greater impact overall.
```

Comment:
```suggestion
Well, the input energy cost and output energy generation are equal now, but the processing energy cost has grown big enough to wipe all the fuel energy gains. Looking at the cause of this high power cost, we see that it is because this recipe takes 30 seconds to craft, while before we were working with 2 or 5 seconds. This difference means that a higher power consumption will have a much greater impact overall.
```